hal/halx86/generic/sysinfo.c

```diff
@@ -24,6 +66,22 @@ HaliQuerySystemInformation(IN HAL_QUERY_INFORMATION_CLASS InformationClass,
 #define REPORT_THIS_CASE(X) case X: DPRINT1("Unhandled case: %s\n", #X); break
 	switch (InformationClass)
 	{
+        case HalQueryAMLIIllegalIOPortAddresses:
```

Comment:
The indentation from the parent block here appears to be 6 spaces. Please either make it 4 (our standard) or 8 (to match the `REPORT_THIS_CASE` lines).

Reply:
Fixed, thanks.
The indentation looked inaccurate because Vadim used the spaces in his code, but in the whole other file, the tabs instead of spaces are used. Now I added the tabs in his code, so now it looks OK.